workflow.md

```diff
@@ -54,6 +54,12 @@ project manager would do it himself). Do not merge your own PRs!
 Please describe in PR what have you done, if it is tested or not, any additional
 information.
 
+## Merging Pull requests
+
+After review by another team member, the teammate merging the PR should merge
+`master` into the Pull request's branch *without* the `--no-ff` option, and then
```

Comment:
@kantp rebase is possible for simple changes. I can't formally describe which changes are simple and which are not. For example, if PR only changes few lines in a single file, it's a simple change which can be rebased on top of master. If PR changes 50 files it's most likely something complex and shouldn't be rebased, because merge will most likely be easier.